Source/WebCore/Modules/WebGPU/GPUQueue.cpp

```diff
@@ -578,7 +578,7 @@ static void* copyToDestinationFormat(const uint8_t* rgbaBytes, GPUTextureFormat
             data[i0 + 1] = rgbaBytes[i + 1];
         }
 
-        sizeInBytes = sizeInBytes * sizeof(float);
+        sizeInBytes = (sizeInBytes / 2) * sizeof(float);
```

Comment:
Maybe this value should be lifted to Line 575 and named `texSizeInBytes` and used in the malloc and change this to `sizeInBytes = texSizeInBytes`?

Reply:
Or `dataSizeInBytes` (naming is hard)

Reply:
I considered that and think it's a good idea. However, this same code occurs about 8 times in this function, I think the risk of accidentally introducing a regression by cleaning it up outweighs the benefit of cleaning it up, as it will be removed entirely once  https://bugs.webkit.org/show_bug.cgi?id=263692  is completed.